src/BuildDacpac/PackageBuilder.cs

```diff
@@ -33,13 +33,17 @@ public void AddReference(FileInfo referenceFile)
             Model.AddReference(referenceFile.FullName);
         }
 
-        public void AddSqlCmdVariable(string variableName)
+        public void AddSqlCmdVariables(string[] variableNames)
         {
             // Ensure that the model has been created
             EnsureModelCreated();
 
-            Console.WriteLine($"Adding sqlcmd variable {variableName}");
-            Model.AddSqlCmdVariable(variableName);
+            foreach (var variableName in variableNames)
+            {
+                Console.WriteLine($"Adding sqlcmd variable {variableName}");
```

Comment:
I know you haven't changed this per se, but would you mind changing `sqlcmd` to be uppercase?